scripts/notify-suppliers-of-new-questions-answers.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+"""
+Send email notifications to supplier users when a question has been answered
+
+Usage: notify-suppliers-of-new-questions-answers.py <stage> <api_token> <mandrill_api_key> [options]
+
+Options:
+    --dry-run                               List notifications that would be sent without sending the emails
+    --exclude-supplier-ids=EXCLUDE-SUPPLIERS   Comma separated list of suppliers IDs to be excluded. This is incase the
```

Comment:
Yeah, I hope it will never happen. If it crops up more than two or three times we can come up with an easier solution.